### PR TITLE
#4882 [Manager] Event log relative minor bugs

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4882)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
@saivirtue I'm testing this locally and looks like the logs are still gone when the page is refreshed 🤔

**Comment by saivirtue** [comment]
> @saivirtue I'm testing this locally and looks like the logs are still gone when the page is refreshed 🤔

My bad. Didn't notice the "notification" word 🤣 
Let me fix it

**Comment by saivirtue** [comment]
> @saivirtue I'm testing this locally and looks like the logs are still gone when the page is refreshed 🤔

Could you give it a shot again?  :)

File: ohara-manager/client/src/store/epics/eventLog/fetchEventLogsEpic.js [hunk 1]
```diff
@@ -48,6 +54,7 @@ export default action$ =>
     ofType(actions.fetchEventLogs.TRIGGER),
     switchMap(() =>
       getLogFromLocalForge$().pipe(
+        defaultIfEmpty(),
```
Comment (saivirtue) [inline ohara-manager/client/src/store/epics/eventLog/fetchEventLogsEpic.js:57]: the `getLogFromLocalForge` observable may return empty if there were no data exist. We need to handle the empty case in order to trigger fetchEventLogs.success action

File: ohara-manager/client/src/store/epics/eventLog/updateNotificationsEpic.js [hunk 2]
```diff
@@ -21,18 +21,22 @@ import { map, catchError, concatMap } from 'rxjs/operators';
 
 import * as actions from 'store/actions';
 import { LOG_LEVEL } from 'const';
+import { infoKey, errorKey } from './const';
 
 const increaseNotification$ = (type, state$) => {
-  const { error, info } = state$.value.entities.eventLogs.notifications.data;
+  const info = parseInt(localStorage.getItem(infoKey)) || 0;
+  const error = parseInt(localStorage.getItem(errorKey)) || 0;
```
Comment (saivirtue) [inline ohara-manager/client/src/store/epics/eventLog/updateNotificationsEpic.js:28]: Use the local storage notification data instead to make sure they are always up-to-date even if user refreshes the page.

**Comment by konekoya** [comment]
@saivirtue Sorry, but it seems like the bug is still there? A page refresh still removes these notification... @@

File: ohara-manager/client/package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
     "lint": "eslint --max-warnings 0 src cypress",
     "lint:ci": "cross-env CI=true eslint --max-warnings 0 src cypress",
     "vet": "yarn audit",
-    "test": "react-scripts test --env=jsdom",
+    "test": "react-scripts test --env=jsdom updateNoti*",
```
Comment (konekoya) [inline ohara-manager/client/package.json:15]: @saivirtue is this a mistake by global search and replace or a real option @@?

Comment (saivirtue) [reply ohara-manager/client/package.json:15]: Yep 😅

**Comment by konekoya** [comment]
@saivirtue Let me know when this PR is ready to review again :)

**Comment by saivirtue** [comment]
> @saivirtue Let me know when this PR is ready to review again :)

Just now 😆

**Pull request merged**